### PR TITLE
feat(claim-dapp): align bar to the middle

### PR DESCRIPTION
**Pull request by Gamaranto**
**Motivation**

This PR aligns the bar to the middle of the dark section of the badge and switches from display the UMA balance to displaying the UMA payoff.

**Testing**

Check a box to describe how you tested these changes and list the steps for reviewers to test.

- [ ]  Ran end-to-end test, running the code as in production
- [ ]  New unit tests created
- [ ]  Existing tests adequate, no new tests required
- [x]  All existing tests pass
- [ ]  Untested

Signed-off-by: Gamaranto <francesco@umaproject.org>

### CODE REVIEW

File: src/components/Redeem.tsx [hunk 1]
```diff
@@ -27,7 +27,8 @@ const Redeem: React.FC<Props> = ({ onCancel }) => {
 
   const logoRef = React.useRef<SVGSVGElement>(null);
   const { balance: optionsBalance } = useOptionsBalance(account);
-  const { balance: umaBalance } = useUMABalance(account);
+  const _umaPayout = Number(optionsBalance) * Number(expirationPayout);
+  const umaPayout = !isNaN(_umaPayout) ? _umaPayout.toFixed(1) : "-";
```
Comment (daywiss) [inline src/components/Redeem.tsx:31]: is this supposed to fail and become '-' if there is no expiration payout yet?

Comment (daywiss) [inline src/components/Redeem.tsx:31]: is this supposed to fail and become '-' if there is no expiration payout yet?

**Review by daywiss — COMMENTED** [review]
just curious about this change to the balance display, it looks like it only will show a value once expired

File: src/components/Redeem.tsx (hunk 1 above)
Comment (Gamaranto) [reply src/components/Redeem.tsx:31]: The `expirationPayout` should always be there in practice, I've added it to use "-" as a fallback when the `optionsBalance` is fetching and might be undefined. 
So TLDR yes, but I haven't added that line for it but rather for the `optionsBalance`

**Review by daywiss — APPROVED** [review]

File: src/components/Redeem.tsx (hunk 1 above)
Comment (Gamaranto) [reply src/components/Redeem.tsx:31]: `optionsBalance` has already been parsed with `ethers.utils.formatEther` and is a string so I would say its safe

**Review by Tulun — APPROVED** [review]

**Pull request merged**